### ISSUE TITLE
Docs: Add shortcut manager docs

### ISSUE THREAD

**Issue by kirszenbaum** [handsontable/handsontable#9153]
To do:
- Add shortcut manager documentation
- Organize the content to make room for future accessibility content

**Comment by warpech** [comment]
I discussed with @wszymanski that the API reference will be generated automatically based on JSDoc.

The guide page should present typical use cases:

- removing keyboard actions built into Handsontable
- replacing shortcuts that trigger actions (by remove and add)
- adding custom keyboard actions

@wszymanski will present snippets that implement these cases.

Typically a programmer will want to do the above in their integration code and/or in their plugin. I am not sure if both cases should be covered (@wszymanski says that the first case is enough).

@AMBudnik do you have ideas for more topics on this page?

**Comment by AMBudnik** [comment]
@warpech add/replace/remove shortcuts seem to cover the subject entirely. 

One question, though, we've been proposing `afterDocumentKeyDown` as in http://jsfiddle.net/903yrmd7/ and I'm wondering if that approach will still be supported in v12? 

Also, could we mention that the creation of custom shortcuts changed (with samples before/after) on our blog? It is an exciting subject. But I wouldn't add that particular comparison in the docs. I remember that setting up a 3-key shortcut was challenging. Showing an example like that would generate a wow-effect in the reader.

Ps. I think that I may have more ideas after the first draft. But feel free to ping me anytime.

**Comment by wszymanski** [comment]
> @warpech add/replace/remove shortcuts seem to cover the subject entirely.
> 
> One question, though, we've been proposing `afterDocumentKeyDown` as in [http://jsfiddle.net/903yrmd7](http://jsfiddle.net/903yrmd7/) and I'm wondering if that approach will still be supported in v12?

The `afterDocumentKeyDown` should work in this way. However, it should be also possible also by adding shortcut by ShortcutManager API.

**Comment by AMBudnik** [comment]
That's a great news @wszymanski ! Thank you for the confirmation.

**Comment by kirszenbaum** [comment]
Done in #9154, closing
